sdk/search/azure-search-documents/tests/test_search_index_client_skillset_live.py

```diff
@@ -28,17 +28,17 @@ class TestSearchSkillset(AzureRecordedTestCase):
     @SearchEnvVarPreparer()
     @search_decorator(schema="hotel_schema.json", index_batch="hotel_small.json")
     @recorded_by_proxy
-    def test_skillset_crud(self, api_key, endpoint):
-        client = SearchIndexerClient(endpoint, api_key)
-        self._test_create_skillset_validation()
-        self._test_create_skillset(client)
-        self._test_get_skillset(client)
-        self._test_get_skillsets(client)
-        self._test_create_or_update_skillset(client)
-        self._test_create_or_update_skillset_if_unchanged(client)
-        self._test_create_or_update_skillset_inplace(client)
-        self._test_delete_skillset_if_unchanged(client)
-        self._test_delete_skillset(client)
+    # def test_skillset_crud(self, api_key, endpoint):
```

Comment:
will this be re-enabled at some point or can it simply be removed ?

Reply:
Service team deprecated some skills on this API version which made the tests fail.

We need to check with them for the plan.

For now, we want to make the vector search available to external users so I disable these tests to unblock.

Reply:
aah gotcha, then just a minor minor nit -> we can mark the test with `pytest.mark.skip(reason="reason")`, otherwise its gtg